data/data.go

```diff
@@ -0,0 +1,19 @@
+package data
+
+import (
+	"encoding/json"
+
+	"github.com/parkmenow/PMN-api/models"
+)
+
+var (
+	users models.UserList
+)
+
+func UserRegistration(reqBody string) string {
+	user := models.User{}
+	json.Unmarshal([]byte(reqBody), &user)
+	users.List = append(users.List, user)
```

Comment:
I believe that there is no need to create a list of users, since we will never fetch some users at anytime. 
After Unmarshalling the json it is sufficient to make a call to a function that handles the writes to the database by passing just the single user json.

Reply:
Yes, You are right! Let's hold this PR till tomorrow. Once the DB is ready will make those specific changes